.travis.yml

```diff
@@ -1,21 +1,7 @@
 language: android
 android:
   components:
-    # Uncomment the lines below if you want to
-    # use the latest revision of Android SDK Tools
     - tools
     - platform-tools
-    - tools
```

Comment:
From https://docs.travis-ci.com/user/languages/android/ "Note that the tools section appears twice on purpose as it’s required to get the newest Android SDK tools." Is my understanding here wrong?

Reply:
Haha, very good point. I actually remember reading something similar. The only reason I removed it is because I saw you removed it over here as well: https://github.com/jshvarts/AppLifecycleDemo/commit/95228bb478562a72e68bd69956618ecec3bf2609
So I thought you prefer not to have it (the build seems to run fine, maybe the info on the Travis doc is outdated..) Anyway, do you want me to re-add it here (and over in the other repo as well, since one commit there touches .travis.yml)?